cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowActionNode.java

```diff
@@ -27,14 +28,44 @@
   private final ScheduleProgramInfo program;
   private final WorkflowActionSpecification actionSpecification;
 
+  /**
+   * Constructs the ACTION node in the {@link Workflow} for MapReduce or Spark program.
+   * @param nodeId the id with which the node is to be created
```

Comment:
There should be an empty line (with a "*") before the first "@param" tag in all Javadocs.